packages/compass-aggregations/src/modules/pipeline-builder/pipeline-mode.ts

```diff
@@ -94,6 +98,7 @@ export const changePipelineMode = (
     track('Editor Type Changed', {
       num_stages,
       editor_view_type: mapPipelineModeToEditorViewType(getState()),
+      connectionInfoAccess,
```

Comment:
```suggestion
      connectionInfoAccess.getCurrentConnectionInfo(),
```

Reply:
ah, I was wondering how did this pass TS, it was supposed to be as a third parameter, not inside 2nd one 🤦 thanks for spotting it!

Reply:
ahh i see now